src/python/pants/engine/target.py

```diff
@@ -1492,6 +1492,21 @@ def __init__(
         )
 
 
+class InvalidSequenceFieldChoiceException(InvalidFieldException):
```

Comment:
See below, I think we can reuse the existing InvalidFieldChoiceException with a small language tweak.

Reply:
Dropped.